### ISSUE TITLE
Updated analysis: LGAT - add additional subtypes

### ISSUE THREAD

**Issue by jharenza** [AlexsLemonade/OpenPBTA-analysis#790]
#### What analysis module should be updated and why?
[LGAT subtyping
](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-LGAT)

#### What changes need to be made? Please provide enough detail for another participant to make the update.
From conversation with Cassie Kline, @adamcresnick, and inspired by this [pediatric LGG review](https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-020-00902-z) and this [pLGG review](https://www.sciencedirect.com/science/article/abs/pii/S1535610820301513), I am updating the subtypes we should add for LGAT tumors. <s>However, I am marking this as `in progress` because I want to discuss with @adamcresnick. I believe there is more about LGAT in the WHO 2016 book (rather than the 2016 publication [here](https://braintumor.org/wp-content/assets/WHO-Central-Nervous-System-Tumor-Classification.pdf)) of which I do not have a copy.</s>

With #631, we currently have the following subtypes for LGAT:
- LGG BRAF fusion
- LGG BRAF V600E
- LGG BRAF WT 

From the Ryall et al., 2020 paper above:
>Together, _KIAA1549-BRAF_ (n = 166), _BRAF_ p.V600E (n = 79), and germline _NF1_ mutations (n = 79) accounted for 68% (n = 324) of tumors. Rare alterations accounted for an additional 17% of cases. These included non-canonical _BRAF_ alterations, such as fusions partnered with genes others than _KIAA1549_ (n = 4), 2 insertion events at position 600 (p.V600ins) and 1 SNV at position 594 (p.D594N) (Figures 2A–2C and 2E). A further 1.3% (n = 6) of cases contained alterations in other direct members of the RAS/MAPK pathway, including three _RAF1_ fusions, two _KRAS_ mutations, and one patient with a short deletion in _MAP2K1_ (Figures 2A–2C and 2E). The next most frequent alterations were those affecting receptor tyrosine kinases (RTKs) (n = 45, 9.4%) (Figures 2A–2C) and included two categories: _FGFR_ and other _RTKs_. Alterations in _FGFR_ most frequently involved _FGFR1/2_ (n = 29, 6.1%) and included _FGFR1-TACC1_ fusions (n = 7, 1.5%), _FGFR1_ tyrosine kinase domain (TKD) duplications (n = 10, 2.1%), _FGFR2_ fusions (n = 5, 1.0%), and hotspot mutations in _FGFR1_ (n = 7, 1.5%) (Figures 2A–2C). Alterations in other _RTKs_ (n = 16, 3.4%) included mutations in _MET_ (n = 5, 1.0%) or _PDGFRA_ (n = 1, 0.2%), as well as fusions involving _ALK_ (n = 2, 0.4%), _ROS1_ (n = 2, 0.4%), and _NTRK2_ (n = 2, 0.4%) (Figures 2A–2C and 2E). Finally, 4.6% (n = 22) of cases contained alterations in genes with seemingly no direct impact on the RAS/MAPK pathway (Figures 2A–2C). These included mutations in _H3F3A_ (n = 4, 0.8%), _IDH1_ (n = 4, 0.8%), and rearrangements involving MYB (n = 6, 1.3%) or _MYBL1_ (n = 5, 1.0%) (Figures 2A–2C). 
We can subtype the first two, but not the third, as we do not have germline data within OpenPBTA.

Figure 2A for reference:
<img width="1068" alt="Screen Shot 2020-10-13 at 3 51 32 PM" src="https://user-images.githubusercontent.com/207034/95909173-07989000-0d6c-11eb-9be1-722759e33cbb.png">

### Suggested updated subtypes (step 1):
- **LGG, NF1**
  - This subtype is characterized by germline _NF1_ variants. We do not have germline data within OpenPBTA, but we do have information about cancer predispositions. This group can be annotated if `NF-1` is present within the `cancer_predispositions` field of the `pbta-histologies.tsv` file. We should also annotate somatic loss of _NF1_ via either missense, nonsense mutation or copy loss. 

- **LGG, KIAA1549-BRAF**
  - contains _KIAA1549-BRAF_ fusion

- **LGG, BRAF V600E**
  - contains BRAF V600E or V599 SNV or non-canonical _BRAF_ alterations such as p.V600ins or p.D594N

- **LGG, other MAPK**
  - contains non-canonical _BRAF_ fusion other than _KIAA1549-BRAF_
  - contains _RAF1_ fusion
  - contains _KRAS_, _NRAS_, _HRAS_, _MAP2K1_, _MAP2K2_, _MAP2K1_, _ARAF_ SNV or indel

- **LGG, RTK**
  - harbors a fusion in _ALK_, _ROS1_, _NTRK1_, _NTRK2_, or _NTRK3_ **or**
  - harbors a _MET_ SNV **or 
  - harbors a _KIT_ SNV **or**
  - harbors a _PDGFRA_ SNV or fusion

- **LGG, FGFR**
  - harbors _FGFR1_ p.N546K, p.K656E, p.N577, or p. K687 hotspot mutations **or** 
  - harbors _FGFR1_ TKD (tyrosine kinase domain tandem duplication) **or**
  - harbors _FGFR1-TACC1_ fusion **or 
  - harbors _FGFR1_ or _FGFR2_ fusions

- **LGG, IDH**
  - harbors an _IDH_ R132 mutation

- **LGG, H3**
  - harbors an _H3F3A_ K28M or G35R/V mutation

- **LGG, MYB/MYBL1**
  - harbors either a _MYB-QKI_ fusion or other _MYB_ or _MYBL1_ fusion

- **LGG, CDKN2A/B** 
  - harbors focal CDKN2A and/or CDKN2B  deletion
  - This is a secondary co-occurring alteration with prognostic significance. 

Since over-expression may be confounded by the inability to distinguish specific exon expression of wildtype vs fusion transcript, we do not need to include an expression filter, per @adamcresnick.

### Step 3:
Per @adamcresnick, we should assess whether we will make a smaller subgroup of **LGG, other MAPK** based on whether all of these will fall within one type of LGG (check the `pathology_free_text_diagnosis` to determine this - ie. are they all PXAs, all pilocytic, etc). If these come from multiple subtypes, we can leave this subtype as is, but if they all come from one, we can lump this in with BRAF as a subtype **LGG, BRAF V600E or MAPK**. Let me know if this makes sense!

#### What input data should be used? Which data were used in the version being updated?
```
pbta-histologies.tsv
pbta-snv-consensus-mutation.maf.tsv.gz
pbta-fusion-recurrently-fused-genes-bysample.tsv
```

#### When do you expect the revised analysis will be completed?
1 week

#### Who will complete the updated analysis?
@kgaonkar6

**Comment by kgaonkar6** [comment]
I just wanted to confirm if we should be checking for p.V600ins or would it be any range of insertion around p600 and should any variant at p599 be captured?

For BRAF we have the following SNVs in lgat biospecimens:
```r
check$HGVSp_Short %>% unique()
[1] "."               "p.V600E"         "p.T599dup"       "p.L410Q"         "p.A598_T599insV"
[6] "p.V600=" 
```

**Comment by jharenza** [comment]
yes, my above note meant to say yes - capture any 599 and 600 variants :)

**Comment by kgaonkar6** [comment]
For H3 subtype , should we include HIST1H3B ?  I didn't find any H3F3A but there is a H3.1 mutatant: 

```r
> consensusMutation[grep("p.K28M",consensusMutation$HGVSp_Short),] %>% dplyr::filter(Tumor_Sample_Barcode %in% lgat_wgs_df$Kids_First_Biospecimen_ID) %>% dplyr::select(Tumor_Sample_Barcode,Hugo_Symbol,HGVSp_Short)
# A tibble: 1 x 3
  Tumor_Sample_Barcode Hugo_Symbol HGVSp_Short
  <chr>                <chr>       <chr>      
1 BS_E5H6CFYT          HIST1H3B    p.K28M  
```

**Comment by jharenza** [comment]
@kgaonkar6 good catch - yes, if exists in either one of those genes, we would add to H3 subgroup. Thanks!

**Comment by jharenza** [comment]
>For the H3.1 samples, I think I will have pathology take a look just in case they should be HGG and since this paper did not mention H3.1, but for now, let's be more specific with the coding to keep track.

@kgaonkar6 to follow up on the above ^, we were able to have the pathology and radiology reports for `BS_E5H6CFYT` reviewed by Mariarita and Angela and I am pasting their response below for the record.

>The mutation present in this tumor H3.1 HIST1H3B K27 M  is a very rare variant but has the same meaning of H3.3. However according to the WHO and C-impact 2 tumor to be defined Diffuse midline glioma H3K27 M has to be diffuse, midline and with the H3.1 or H3.3 histone mutation.
1)This tumor is LOBAR; left frontal mass
2)The radiology seems to be supportive of a fairly well circumscribed tumor
3)Ganglioglioma with H3.3 or H3.1 have been described
4)The grading of these tumors is unknown  because ARE NOT DIFFUSE MIDLINE GLIOMAS
5)The prognosis of the tumors (ependymomas, gangliogliomas and PA) with histone mutation but without the features of diffuse midline glioma is unknown at the present.
WOULD BE IMPORTANT FOR US TO REVIEW THE SLIDES

It looks like this is a rare event and we should keep this tumor sample as is. Previously, we had annotated it as DMG, H3 mutant while we awaited radiology slides, as it came up with the HGAT module due to harboring a histone mutation. With #807, we will be removing LGAT which harbor histone variants, so the double subtyping currently seen in the pathology subtyping module for this sample will be resolved.
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/3f8baa5686f9f5e771188a60a3bbb01cff3dc495/analyses/molecular-subtyping-pathology/03-incorporate-pathology-feedback.Rmd#L32

I am just double confirming with @adamcresnick and Cassie Kline that we want to keep the ganglioglioma samples within `LGG, H3.1` and not have a separate `GNG, H3.1` subtype.

**Comment by kgaonkar6** [comment]
For CDKN2A/B should we only used copy_number <=1 ? Here's the distribution of copy_number

```r
> cnv_focal_anno_lgat %>%
   dplyr::filter(gene_symbol %in% c("CDKN2A","CDKN2B")) %>%
   dplyr::pull(copy_number) %>%
   table()

  0   1   2   3 
  2  16 134   6 
```

**Comment by kgaonkar6** [comment]
@jharenza  can I get more guidance with respect to this description within **LGG, FGFR** "harbors FGFR1 TKD (tyrosine kinase domain tandem duplication". 

Would we look for "tandem duplication" in manta file where it's annotated as "<DUP:TANDEM>" ? 

```r
> check[,c("Gene.name","ALT","Kids.First.Biospecimen.ID.Tumor")] %>% as.data.frame()
   Gene.name          ALT Kids.First.Biospecimen.ID.Tumor
1      FGFR1 <DUP:TANDEM>                     BS_D6XHKZDZ
2      FGFR1 <DUP:TANDEM>                     BS_8JFMP1T1
3      FGFR1 <DUP:TANDEM>                     BS_3108RS7G
4      FGFR1 <DUP:TANDEM>                     BS_M2MSWKBC
5      FGFR1 <DUP:TANDEM>                     BS_ETTZM63Y
6      FGFR1 <DUP:TANDEM>                     BS_MW16Y7ME
7      FGFR1 <DUP:TANDEM>                     BS_W6B48DD5
8      FGFR1 <DUP:TANDEM>                     BS_YKQP4SBX
9      FGFR1 <DUP:TANDEM>                     BS_T99M7842
10     FGFR1 <DUP:TANDEM>                     BS_4G9Y2VC6
11     FGFR1 <DUP:TANDEM>                     BS_V79WFJ6N
12     FGFR1 <DUP:TANDEM>                     BS_R726FKHP
13     FGFR1 <DUP:TANDEM>                     BS_C3X4AEK5
14     FGFR1 <DUP:TANDEM>                     BS_KSHETTQC
15     FGFR1 <DUP:TANDEM>                     BS_E5B2YZJH
16     FGFR1 <DUP:TANDEM>                     BS_MW6YMRBJ

```

From consensus_seg we would get the info if it's copy_number but I'm not sure if it will be specifically "tandem duplication"

**Comment by jharenza** [comment]
> For CDKN2A/B should we only used copy_number <=1 ? Here's the distribution of copy_number
> 
> ```r
> > cnv_focal_anno_lgat %>%
>    dplyr::filter(gene_symbol %in% c("CDKN2A","CDKN2B")) %>%
>    dplyr::pull(copy_number) %>%
>    table()
> 
>   0   1   2   3 
>   2  16 134   6 
> ```

yes, I think we can start with that

**Comment by jharenza** [comment]
> @jharenza can I get more guidance with respect to this description within **LGG, FGFR** "harbors FGFR1 TKD (tyrosine kinase domain tandem duplication".
> 
> Would we look for "tandem duplication" in manta file where it's annotated as "DUP:TANDEM" ?
> 
> ```r
> check[,c("Gene.name","ALT","Kids.First.Biospecimen.ID.Tumor")] %>% as.data.frame()
>   Gene.name          ALT Kids.First.Biospecimen.ID.Tumor
> 1     FGFR1 <DUP:TANDEM>                     BS_3108RS7G
> 2     FGFR1 <DUP:TANDEM>                     BS_ETTZM63Y
> 3     FGFR1 <DUP:TANDEM>                     BS_R726FKHP
> 4     FGFR1 <DUP:TANDEM>                     BS_MW6YMRBJ
> ```
> 
> From consensus_seg we would get the info if it's copy_number>2 but I'm not sure if it will be specifically "tandem duplication"
> @jharenza I updated the df ^ with only lgat samples

From [this](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3951336/) and [this](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5039033/) publication, it looks like the tandem duplication breakpoints in `FGFR1` happen between exons 10-18, so I suggest that we take these tandem dup events and overlap with those exons to make sure these are happening in the kinase domain. 

Looking at the copy number is also a good idea, though, I would not exclude the samples if we do not see the CNV changes, because I think we still have work to do on CN, but please do add it to the notebook.

**Comment by kgaonkar6** [comment]
Seems like in the 01 script I only looked for 599, 600 **OR** SNV/indels overlapping tyrosine kinase domain in BRAF but isn't documented in this ticket so adding it here:

 https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/f85831fe7e405b1c6b6c43274cabcc796b7bd3b1/analyses/molecular-subtyping-LGAT/01-subset-files-for-LGAT.Rmd#L146-L156

```r
consensusMutationSubset[grep("BRAF",consensusMutationSubset$Hugo_Symbol),"HGVSp_Short"] %>% unique()
# A tibble: 4 x 1
  HGVSp_Short    
  <chr>          
1 p.V600E        
2 p.T599dup      
3 p.A598_T599insV
4 p.V600= 
```

**Comment by jharenza** [comment]
closed via #925
